docs/TempUG.md

```diff
@@ -33,16 +33,16 @@ This project is based on the [AddressBook-Level3 project](https://se-education.o
 
 1. Ensure you have Java 11 or above installed in your Computer. You can check by opening a command terminal and typing `java -version`.
 
-1. Download the latest `tutormate.jar` from [here](https://github.com/AY2324S1-CS2103T-T11-3/tp/releases).
+2. Download the latest `tutormate.jar` from [here](https://github.com/AY2324S1-CS2103T-T11-3/tp/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your TutorMate.
+3. Copy the file to the folder you want to use as the _home folder_ for your TutorMate.
 
-1. Open a command terminal, navigate into the folder you put the jar file in using the change directory command `cd`, and use the `java -jar tutormate.jar` command to run the application.<br>
+4. Open a command terminal, navigate into the folder you put the jar file in using the change directory command `cd`, and use the `java -jar tutormate.jar` command to run the application.<br>
```

Comment:
since you used the phrase "home-folder" previously, why not reuse it here?

Reply:
Make the necessary changes in the recent commit, thank you ! ◡̈